gpbft/api.go

```diff
@@ -38,9 +38,8 @@ type Chain interface {
 type Network interface {
 	// Returns the network's name (for signature separation)
 	NetworkName() NetworkName
-	// Sends a message to all other participants.
-	// The message's sender must be one that the network interface can sign on behalf of.
-	Broadcast(msg *GMessage)
+	// Requests that the message is signed and broadcasted, it should also be delivered locally
+	RequestBroadcast(msg *GMessage)
```

Comment:
Changing semantics is fine. I don't think we need to rename this method if the old behaviour is gone.

Reply:
I went with this approach because naming it `Broadcast` leads to a collision among the interfaces used by `Host`.

Reply:
I don't think you need that method in the adversary host, though.

Reply:
You are right, this is no longer necessary. Thanks!